frontend/src/components/common/RenderMultiSelect.js

```diff
@@ -44,6 +46,7 @@ export const RenderMultiSelect = (props) => (
         getPopupContainer={() => document.getElementById(props.id)}
         placeholder={props.placeholder}
         id={props.id}
+        onSearch={props.onSearch}
         {...props.input}
```

Comment:
did you test to make sure it didn't break the previous multi selects that aren't passing in an onSearch function?